### DISCUSSION TITLE
GSoC 2022: The proposal submission period will start

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7203] [Announcements]
Hello every contributor:

Thanks for interesting for Robolectric's GSoC 2022 program. [This program will enter proposal submission period: April 4 - 19 (1800 UTC)](https://opensource.googleblog.com/2022/04/google-summer-of-code-2022-contributor-applications-now-open.html):

> Contributor Proposal Submission period: April 4 -19 (1800 UTC)
>
> Starting Monday, April 4 at 1800 UTC, contributors will be able to submit their proposals via the program website. Each contributor may submit up to 3 proposals to the program. They can submit multiple proposals to the same organization, or spread their proposals across multiple organizations.

It's recommended to prepare your proposal recent days, and discuss it with project's potential mentors that you have interested in. Feel free to ask any question at related project's Discussion page.

It's also recommended to participate in [easy/starter issue](https://github.com/robolectric/robolectric/issues?q=is%3Aissue+is%3Aopen+label%3A%22starter+issue%22) of Robolectric, and it will help you to understand what Robolectric is and how it works.

Hope everyone has a great journey with Robolectric's GSoC 2022 program.

Thanks.

**Comment by hellosagar** [comment]
My proposal for feedback purpose from mentors before actually submitting to the gsoc portal (@utzcoz @hoisie @JuliaSullivanGoogle @dennismeng)

https://docs.google.com/document/d/1oFBN4exBqnYAR_YkWsy290uhC2aYPK8ux0OoLwnO30w/edit?usp=sharing

**Reply by hellosagar** [reply]
If you think something is missing or can be improved 

Love to have your feedback

**Reply by hellosagar** [reply]
Submitted!
![Screenshot 2022-04-18 at 12 26 23 PM](https://user-images.githubusercontent.com/50016799/163768906-c99cb7a6-d67f-4807-962d-08fe4369aac9.png)
Hope for the best.

If I'll get selected I will try to contribute as much as possible to Robolectric and learn to become a better engineer 🤞🤞🤞

**Comment by hellosagar** [comment]
Mentors, what this phrase means precisely?

<img width="453" alt="Screenshot 2022-05-20 at 12 02 33 AM" src="https://user-images.githubusercontent.com/50016799/169374491-ecdc66bb-ab85-4376-bb1a-fa946826be75.png">

**Reply by utzcoz** [reply]
We have received Google's email for project's acceptance status. The accepted contributors list will be released by Google in recent hours, at least inner one day in my time zone. Please keep patience, and wait Google's official announcement.
